### PR TITLE
[Reviewer: Rob] Adjust event levels to fit new guidelines

### PR DESCRIPTION
**Pull request by AlexHockey**
This PR adjusts the levels of our existing SAS events to conform to the new guidelines. Please can you review?

This PR has the following themes:

* Many sprout mainline events are quite useful but were logged at a low level. These have become level 80. 
* Most Ralf events have been reduced to level 40, because they can be raised very frequently and are classed as spammy. 
* In some places there were events that were good candidates for level 80 events, but the language was not appropriate (e.g. they referred to memcached). I have tweaked the language in these cases. 

I have loaded some exports into SAS with the new bundle and checked they look vaguely sensible, but I am going to hold off further testing until I have added the new events we are planning to add.

### CODE REVIEW

File: clearwater_sas_resource_bundle.yaml [hunk 1]
```diff
@@ -468,15 +469,15 @@ events:
 
   0x810043:
     summary: 'Request authenticated successfully'
-    level:   40
+    level:   80
 
   0x810044:
-    summary: 'Challenge the request. Request an AKA authentication vector from memcached'
-    level:   40
+    summary: 'Challenge the request using AKA authentication'
```
Comment (rkd-msw) [inline clearwater_sas_resource_bundle.yaml]: These seem unusually-phrased to me - everything else is descriptive, but these are imperative. "Request will be challenged..."?

File: clearwater_sas_resource_bundle.yaml [hunk 2]
```diff
@@ -468,15 +469,15 @@ events:
...
+    level:   80
 
   0x810045:
-    summary: 'Challenge the request. Request a DIGEST authentication vector from memcached'
-    level:   40
+    summary: 'Challenge the request using DIGEST authentication'
```
Comment (rkd-msw) [inline clearwater_sas_resource_bundle.yaml]: DIGEST isn't an acronym (unlike AKA), so it should be Digest.

File: clearwater_sas_resource_bundle.yaml [hunk 3]
```diff
@@ -798,8 +803,8 @@ events:
     level:   80
 
   0x8200A0:
-    summary: "No HSS configured - returning default server name if subscriber exists"
-    level:   40
+    summary: "No HSS configured - returning default server name"
```
Comment (rkd-msw) [inline clearwater_sas_resource_bundle.yaml]: I think the "if subscriber exists" is still useful information (it's probably better to raise a different SAS log in that case, but until we do that, best not to lose the information).

**Comment by rkd-msw** [comment]
Looks good - I've spotted a few minor issues with log text while reading through (only one introduced by you).

I don't have a good feel on what's useful for Ralf, due to the dearth of available CDFs to test with - I'm happy to drop everything to 40, but we should revisit in future.

File: clearwater_sas_resource_bundle.yaml (hunk 3 above)
Comment (AlexHockey) [inline clearwater_sas_resource_bundle.yaml]: I was correcting an error here (I checked the code, and after we raise this log homestead just returns an HTTP response with the default server without checking if the subscriber exists).

Comment (rkd-msw) [inline clearwater_sas_resource_bundle.yaml]: Sounds good.

**Pull request merged**